packages/lib-classifier/package.json

```diff
@@ -20,8 +20,8 @@
     "eslint": "eslint ./src --fix | snazzy",
     "prepare": "yarn build:cjs && yarn build:es6",
     "start": "PANOPTES_ENV=production npm run dev",
-    "test": "NODE_PATH=. NODE_ENV=test mocha --exit --config ./test/.mocharc.json ./.storybook/specConfig.js  \"./{src,test}/**/*.spec.js\"",
-    "test:ci": "NODE_PATH=. NODE_ENV=test mocha --exit --config ./test/.mocharc.json ./.storybook/specConfig.js  --reporter=min \"./{src,test}/**/*.spec.js\"",
+    "test": "NODE_ENV=test vitest run",
+    "test:ci": "NODE_ENV=test vitest run --reporter=basic --silent='passed-only' --hideSkippedTests",
```

Comment:
Console logs from the classifier are thousands of lines, so `test:ci` is configured to silence console logs unless a test fails. Also no need to print out the name of spec files that are entirely skipped.